module/experiment.js

```diff
@@ -2,13 +2,20 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { Variant } from './variant'
-import { setExperimentVariant } from './actions'
+import { addExperiment } from './actions';
+
+const originalVariant = 'original'
 
 export function Selector (props) {
-	const { variant, name, children } = props
-	let chosenOne
+	const { variant, name, children, addExperiment } = props
+	let searchedVariant = variant, chosenOne
+
+    if (variant == null) {
+        searchedVariant = originalVariant
+        addExperiment(name, searchedVariant)
```

Comment:
there's no need to add the experiment in the store here. Just render the original if there is not an experiment setup.

Reply:
ok, I'll do that